Codeinwp/neve
# chore: add control to hide title on archive pages

### Summary

Add's option to hide archive titles

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Install this patch
- Go to customizer->layout->blog/archive, there will be a new option to "Disable Title"
- This should disable titles on all archive types, including category, tags, and custom post type archives titles


Closes #3464
